osmosis-1/osmosis.zone_assets.json

```diff
@@ -1566,6 +1566,13 @@
       "path": "transfer/channel-1411/umpwr",
       "osmosis_main": false,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "xpla",
+      "base_denom": "axpla",
+      "path": "transfer/channel-9/axpla",
```

Comment:
this would actually be channel-1634 (the osmosis-side channel)

Reply:
Channel changed.